### DISCUSSION TITLE
How to define path to pyrightconfig.json?

### DISCUSSION THREAD

**Discussion by mgzenitech** [microsoft/pyright#8282] [Q&A] [answered]
My use case is that we put all our config files for linters,formatters,etc under specific folder:
```
.internal/configs
```

How do I setup the VSCode to use `.internal/configs/pyrightconfig.json` instead of creating one in root? For workaround purposes I've created a symlink, but that's suboptimal as we'd like to be able to move this file.

**Comment by erictraut** [comment, marked as answer]
When used with a language server or VS Code extension, pyright always looks in the root of the project for a `pyrightconfig.json` or `pyproject.toml` file. You could create a minimal `pyrightconfig.json` or `pyproject.toml` that "extends" another config file located elsewhere in your project, but this minimal config file would need to be in the root of your project.

**Reply by mgzenitech** [reply]
ok, looks like we can use extens in this case
```
[tool.pyright]
extends = ".internal/configs/pyright.json"
```

and then move everything to json file.
